src/Runner.Worker/ActionManager.cs

```diff
@@ -610,6 +610,7 @@ private async Task BuildActionContainerAsync(IExecutionContext executionContext,
                     {
                         NameWithOwner = repositoryReference.Name,
                         Ref = repositoryReference.Ref,
+                        Path = repositoryReference.Path,
```

Comment:
nit: consider deleting the comment on this method (line 595)

Reply:
`my-org/my-repo/my-folder/my-subfolder@v1`

- NameWithOwner: my-org/my-repo
- Ref: v1
- Path my-folder/my-subfolder